x-pack/plugins/apm/public/components/app/trace_link/index.tsx

```diff
@@ -25,6 +26,11 @@ export function TraceLink() {
     query: { rangeFrom, rangeTo },
   } = useApmParams('/link-to/trace/{traceId}');
 
+  const { start, end } = useTimeRange({
+    rangeFrom: rangeFrom || new Date(0).toISOString(),
+    rangeTo: rangeTo || new Date().toISOString(),
```

Comment:
Does the fallback range query have any effect? It's looking at 1970 up until now?

Reply:
Ah.. I suppose it's because it's now required?

Reply:
Yes - this is for the deeplink, where we don't require the rangeFrom/rangeTo parameter, and I don't want to break existing links